src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -456,7 +473,7 @@ def _dss_url(self, file):
         return dss_url
 
     def return_response(self):
-        if config.disable_multipart_manifests or self.format == 'bdbag':
+        if config.disable_multipart_manifests or self.format in('bdbag' , 'full'):
```

Comment:
```suggestion
        if config.disable_multipart_manifests or self.format in ('bdbag', 'full'):
```
Add a space between `in` and `('bdbag'` and and remove the space after `bdbag` and youn should be ready for approval.